server/athenian/api/preloading/cache.py

```diff
@@ -73,10 +79,19 @@ def df(self) -> pd.DataFrame:
         assert self._df is not None, "Need to await refresh() at least once."
         return self._df
 
+    @property
+    def filtering_clause(self) -> sa.sql.ClauseElement:
+        """Return the filtering clause used for refreshing the data."""
+        return self._filtering_clause
+
+    @filtering_clause.setter
+    def filtering_clause(self, fc: sa.sql.ClauseElement):
+        self._filtering_clause = fc
```

Comment:
It's interesting that the docstring is missing here but `flake8` passes without problems.

Reply:
Maybe because it excludes properties as they're self-explanatory? 🤷‍♂️